tests/Agent/IntegrationTests/IntegrationTests/IntegrationTests.csproj

```diff
@@ -26,10 +26,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
-    <PackageReference Include="Selenium.Support" Version="2.53.0" />
-    <PackageReference Include="Selenium.WebDriver" Version="2.53.0" />
+    <PackageReference Include="Selenium.Support" Version="3.141.0" />
+    <PackageReference Include="Selenium.WebDriver" Version="3.141.0" />
     <PackageReference Include="SharpZipLib" Version="1.1.0" />
-    <PackageReference Include="WebDriver.IEDriverServer.win32" Version="2.53.0" />
+    <PackageReference Include="WebDriver.IEDriverServer.win32" Version="3.150.1" />
```

Comment:
In the past we didn't typically update the test dependencies.  I like that these were updated!